SpatialGDK/Source/SpatialGDK/Public/Tests/SpatialView/ComponentTestUtils.h

```diff
@@ -21,7 +21,7 @@ inline ComponentData CreateTestComponentData(const Worker_ComponentId Id, const
 	ComponentData Data{ Id };
 	Schema_Object* Fields = Data.GetFields();
 	Schema_AddDouble(Fields, EntityComponentTestUtils::TEST_DOUBLE_FIELD_ID, Value);
-	return Data;
+	return MoveTemp(Data);
```

Comment:
No need to move here. It will be elided in all likelihood and otherwise the move is implicitly added.